lib/pymedphys/docs/lib/howto/get-started.rst

```diff
@@ -6,21 +6,61 @@ Install Python
 ==============
 
 In order to make use of the PyMedPhys library, you'll need Python installed on
-your workstation. On Windows we recommend that you install the
-Anaconda Python distribution. Download the latest Anaconda
-version from `here <https://www.anaconda.com/products/individual#Downloads>`__.
+your workstation. Below are some recommended instructions for installing Python
+based upon your OS.
 
-.. note::
+Windows 10 or 11
+----------------
 
-    When installing Anaconda make sure to install it for your user only, and
-    tick the option "add to path".
+Open up a command prompt and type:
 
-.. image:: ../../img/add_anaconda_to_path.png
+.. code:: bash
+
+    python
+
+If python isn't already installed this will open the Windows store. At this
```

Comment:
I'm curious, does this way of installing python require admin rights? Just thinking that for our clinical physicists this could be the first stumbling block...

Reply:
I'm actually not sure... I suspect updating the path length beyond 260 would certainly need admin rights... and this Windows Store method installs Python in a "very nested" directory. So much so that Windows store method with a 260 character path limit means that some of pymedphys' dependencies can't install.

I do notice though that this max path limit is recommended within the Python install docs:

https://docs.python.org/3/using/windows.html#removing-the-max-path-limitation

Reply:
Yep, fair enough. Let's leave it for now. But I'll try to run through this at some point with one of our clinical physicists and see how far they get. Perhaps we'll just have to add at the top that this guide assumes they are working on a device where they have admin rights.

Reply:
> But I'll try to run through this at some point with one of our clinical physicists and see how far they get

That'd be amazing

> Perhaps we'll just have to add at the top that this guide assumes they are working on a device where they have admin rights.

I think if there is a simple approach that doesn't need admin rights we should opt to go that way. It certainly is possible (I didn't have admin rights back at RCCC, I did get the path limit bumped by IT though).

Reply:
Hi @pchlap,

When you do get someone to run through it, it appears the Windows 10 store install approach doesn't add python scripts to the path. So the pymedphys CLI won't be available. PyMedPhys can still be called by running `python -m pymedphys --help`. 

It might be worth us having a "test to make sure the pymedphys CLI is working" section, and if it's not, know that you can use `python -m pymedphys etc...` or you can use pymedphys directly by adding Scripts to the path (provide instructions on how to do this).